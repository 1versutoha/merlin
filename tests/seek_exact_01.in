["tell","source","let a = 4\nlet b = 6\nlet c = 7"] 
["tell","end"]
["seek","position"]
["seek","exact",{"line":3,"col":9}]
["seek","exact",{"line":2,"col":9}]
["seek","exact",{"line":3,"col":9}]
["seek","maximize_scope"]
