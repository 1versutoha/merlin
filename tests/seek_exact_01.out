["return",null]
["return",{"line":3,"col":9}]
["return",{"line":3,"col":9}]
["return",{"line":3,"col":9}]
["return",{"line":2,"col":9}]
["return",{"line":3,"col":9}]
["return",{"line":3,"col":9}]
